libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -112,7 +112,9 @@ for FrSky SPort Passthrough
 #define ATTIANDRNG_PITCH_OFFSET     11
 #define ATTIANDRNG_RNGFND_OFFSET    21
 // for fair scheduler
-#define TIME_SLOT_MAX               11
+#define TIME_SLOT_MAX               12
+#define SPORT_PACKET_SIZE           9U
+#define STUFF_MASK                  0x20
```

Comment:
Don't forget anything in this sort of header ends up everywhere in the code :-)

Reply:
yeah, would moving them inside the class as const make it better? (perhaps in another PR)

Reply:
Worth a try as a `static const`.  Check the build size before/after - should be the same.